ksqldb-engine/src/main/java/io/confluent/ksql/query/QueryBuilder.java

```diff
@@ -579,7 +579,7 @@ public static Map<String, Object> buildStreamsProperties(
         ? queryId.get().toString()
         : "query";
 
-    if (config.getBoolean(KsqlConfig.KSQL_SHARED_RUNTIME_ENABLED)) {
+    if (config.getBoolean(KsqlConfig.KSQL_SHARED_RUNTIME_ENABLED) && type.equals("query")) {
```

Comment:
can we avoid hardcoding the `query`? possibly via some global variable.config

Reply:
+1, let's make it a constant in `QueryApplicationId` maybe?

Reply:
Granted, there are a few other places (that I know of) where we would probably want to switch over to using constants, so we could also just merge this one as-is and make a pass to replace hardcoded strings with a constant in a followup PR

Reply:
Its okay the build didn't kick off so I need to push again anyways